taprpc/priceoraclerpc/price_oracle.proto

```diff
@@ -0,0 +1,105 @@
+syntax = "proto3";
+
+package priceoraclerpc;
+
+option go_package = "github.com/lightninglabs/taproot-assets/taprpc/priceoraclerpc";
+
+service PriceOracle {
+    /* tapcli: `rfq priceoracle ratetick`
+    QueryRateTick queries the rate tick for a given transaction type, subject
+    asset, and payment asset. The rate tick is the exchange rate between the
+    subject asset and the payment asset.
+    */
+    rpc QueryRateTick (QueryRateTickRequest) returns (QueryRateTickResponse);
+}
+
+// TransactionType is an enum representing the type of transaction.
+enum TransactionType {
+    // PURCHASE indicates a purchase transaction.
+    PURCHASE = 0;
+
+    // SALE indicates a sale transaction.
+    SALE = 1;
+}
+
+// RateTick is the internal unit used for asset conversions. A tick is 1/10000th
+// of a currency unit. It gives us up to 4 decimal places of precision (0.0001
+// or 0.01% or 1 bps). As an example, if the BTC/USD rate was $61,234.95, then
+// we multiply that by 10,000 to arrive at the usd_rate_tick:
+// $61,234.95 * 10000 = 612,349,500. To convert back to our normal rate, we
+// decide by 10,000 to arrive back at $61,234.95.
+message RateTick {
+    // rate is the exchange rate between the subject asset and the payment
+    // asset.
+    uint64 rate = 1;
+
+    // precision is the number of decimal places in the exchange rate.
+    uint32 precision = 2;
```

Comment:
Why's this required? Just for display purposes somewhere?

Reply:
Will remove. We can always add it back later if we need greater precision.